packages/repository/test/unit/decorator/relation.decorator.unit.ts

```diff
@@ -100,33 +95,9 @@ describe('relation decorator', () => {
         target: () => Address,
         keyTo: 'someForeignKey',
       });
-      expect(jugglerMeta).to.eql({
```

Comment:
ditto

Reply:
The original tests were added and changed accordantly